change/@fluentui-azure-themes-a88a46a2-005b-456c-9295-49ecb3d78029.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32031&commitId=52381bc99d1d54e864d651caffc3d6f4006ce662&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.normal.chromium.png|3| Changed|

</details>


  <details><summary><b>Card Converged - Selectable</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.normal.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.selected.chromium.png|0| Removed|

</details>


  <details><summary><b>SwatchPicker Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.spacing.default.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>